test/e2e/wallet-helpers/api-helpers.ts

```diff
@@ -0,0 +1,62 @@
+import { WebDriver } from 'selenium-webdriver'
+export class APIHelper {
+  private driver: WebDriver
+
+  constructor(driver: WebDriver) {
+    this.driver = driver
+  }
+
+  async generateRecoveryPhrase() {
+    return await this.driver.executeScript<string>(async () => {
+      // @ts-ignore
+      const { recoveryPhrase } = await window.client.request('admin.generate_recovery_phrase', null)
```

Comment:
chore (non-blocking): suggest adding a index.d.ts file to tell typescript that the window object has this client prop rather than requiring an ignore everywhere.

```
interface Window {
  client?: JSONRPCClient;
}
```